README.md

```diff
@@ -33,7 +33,8 @@ var rdbStore = new RDBStore({
     buffer: 50,
     max: 1000,
     timeout: 20,
-    timeoutError: 1000
+    timeoutError: 1000,
+    silent: false
```

Comment:
You're probably right. I guess my original thought was to show the default value in the example